app/controllers/users/sessions_controller.rb

```diff
@@ -1,27 +1,29 @@
 # frozen_string_literal: true
 
 class Users::SessionsController < Devise::SessionsController
-  # before_action :configure_sign_in_params, only: [:create]
+  before_action :configure_sign_in_params, only: [ :create ]
+
+  layout "login"
 
   # GET /resource/sign_in
-  # def new
-  #   super
-  # end
+  def new
```

Comment:
You do not really need to uncomment such actions. They will work the same. You can even delete them.